book/P1C5_eha.qmd

```diff
@@ -12,159 +12,456 @@ abstract: TODO (150-200 WORDS)
 
+Per @eq-ms-prodint-approx all we need for estimation of transition probabilities is an estimate of the transition specific cumulative hazards $H_{\ell e}(\tau)$.
+Luckily, similar consideration as for competing risks in @sec-aalen-johanson apply, replacing the cause-specific hazards (@eq-cs-discrete-hazard) with transition-specific hazards
+<!--  -->
+$$
+h^{d}_{\ell e}(t_{(k)}):= \frac{d_{\ell e,t_{(k)}}}{n_{\ell; t_{(k)}}},
+$$
+<!--  -->
+where
 
-## Recurrent Events {#sec-recurrent-events}
+- $d_{\ell e,t_{(k)}}$ is now the number of observations with transition $\ell \rightarrow e$ at time $t_{(k)}$ and
+- $n_{\ell; t_{(k)}} = |\mathcal{R}_{\ell;t_{(k)}}|$ is the number of subjects in state $\ell$ at time $t_{(k)}$.
 
-The *recurrent events* setting is defined by the same event type occurring multiple times.
-While the recurrent events setting can be viewed as a special case of the multi-state setting (@sec-multi-state), we treat it here separately as it can also be approached as single event data with correlation.
-Some examples of recurrent events include recurrence of infectious diseases, episodes of epilepsy, and replacements of machine parts.
+Thus, risk set $\mathcal{R}_{\ell;t_{(k)}}$ excludes subjects who entered a terminal state or were censored before time $t_{(k)}$, but also subjects who are not in state $\ell$ at that time (either because they did not reach it yet or transition to another state occurred beforehand).
+
+The cumulative transition-specific hazards thereby follow as
+<!--  -->
+$$
+H_{NA,\ell e}(\tau) = \sum_{k:t_{(k)}\leq \tau} h^d_{\ell e}(t_{(k)}) = \sum_{k:t_{(k)}\leq \tau}\frac{d_{\ell e,t_{(k)}}}{n_{\ell; t_{(k)}}},
+$$
+<!--  -->
+
+and transition probabilities are obtained via @eq-ms-prodint-approx as
+<!--  -->
+$$
+\hat{\mathbf{P}}(\zeta, \tau) \approx \prod_{j=1}^{J}(\mathbf{I} + \Delta\hat{\mathbf{H}}_{NA,\ell e}(t_{(j)})),
+$$ {#eq-ms-aj-trans-prob}
+<!--  -->
 
-@tbl-cgd-data shows data of chronic granolotomous disease (CGD) for two subjects.
-Note the start-stop notation of the data ($t_{start}, t_{stop}$ respectively) that indicates the time in which a subject enters the risk set for the k*th* event number.
-The column $\delta$ indicates whether the subject experienced the $k$th outbreak or was censored.
-For example, Subject 1 (first three rows) experienced the first outbreak (first row) after 219 days, at which time they entered the risk set for event number 2.
-The second event (second row) was experienced at 373 days and the subject was then censored for the third event (third row) after 414 days.
-This notation is equivalent to a multi-state representation of the recurrent events process, where each event number represents a different state.
-In @fig-recurrent-events, right panel, this is refered to as "Clock forward" approach.
+For illustration, consider again the `prothr` data set (@tbl-ms-data-prothr), with possible transitions summarized in @fig-multi-state-prothr-states.
```

Comment:
Slightly more lead-in to the example would be good. Kind of abruptly jumps at the moment